Projects/OpenTK.Android/Xamarin.Legacy.OpenTK.targets

```diff
@@ -0,0 +1,5 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
It might be worth making this file `Xamarin.Legacy.OpenTK.props` instead of `.targets`.

So users could remove the `@(ProguardConfiguration)` item in their project file. Just in case there would be some reason they need to?